code/game/objects/structures/flora.dm

```diff
@@ -31,6 +31,12 @@
 	max_integrity = 500
 	layer = ABOVE_FLY_LAYER
 	var/log_amount = 10
+	var/y_coverage = 2
+	var/x_coverage = 0
```

Comment:
Either a proc to change this, with a signal registered by the component, or just a proc ran on `Initialize()` that sends these values and can be redefined by children objects, instead of variables?

Reply:
Did that, doesn't look at lot better though.